subgraphs/_reference_/src/prices/index.ts

```diff
@@ -132,7 +110,7 @@ export function getUsdPrice(
   tokenAddr: Address,
   amount: BigDecimal
 ): BigDecimal {
-  let tokenPrice = getUsdPricePerToken(tokenAddr);
+  const tokenPrice = getUsdPricePerToken(tokenAddr);
 
   if (!tokenPrice.reverted) {
     return tokenPrice.usdPrice.times(amount).div(tokenPrice.decimalsBaseTen);

```

Comment:
CustomPrice.usdPrice should really be normalized to USD price with proper decimals. There is no reason it needs to be done after the fact.

After that you can also remove decimalsBaseTen

Reply:
Yeah, that makes more sense.